compiler/rustc_hir_typeck/src/fn_ctxt/adjust_fulfillment_errors.rs

```diff
@@ -864,3 +864,18 @@ fn is_iterator_singleton<T>(mut iterator: impl Iterator<Item = T>) -> Option<T>
         (first, _) => first,
     }
 }
+
+#[derive(Copy, Clone, PartialEq, Eq, Debug)]
+pub enum ParamTerm {
```

Comment:
lol, I moved this into middle because someone asked me to. I'd probably just drop this commit, imo.

Reply:
Oh lol. Okay.